MAINTAINERS

```diff
@@ -31,6 +31,78 @@ to appreciate the absence of bugs, the slow but steady improvement in stability,
 or the reliability of a release process. But those things distinguish a good
 project from a great one.
 """
+    [Rules.adding-maintainers]
+
+        title = "How are maintainers added?"
+
+        text = """
+Maintainers are first and foremost contributors that have shown they are
+committed to the long term success of a project. Contributors wanting to
+become maintainers are expected to be deeply involved in contributing code,
+pull request review, and triage of issues in the project for more than three
+months.
+
+Just contributing does not make you a maintainer, it is about building trust
+with the current maintainers of the project and being a person that they can
+depend on and trust to make decisions in the best interest of the project.
+
+Maintainers are assigned per project (repository). Being a maintainer in
+one project does not automatically make you a maintainer in other projects.
+
+Periodicly, the existing maintainers curate a list of contributors that have
+shown regular activity on the project over the prior months. From this
+list, maintainer candidates are selected and proposed on the maintainers
+mailing list.
+
+After a candidate has been announced on the mailing list, the existing
+maintainers are given five business days to discuss the candidate, raise
+objections and cast their vote. Candidates must be approved by over 66% of the
+current maintainers by adding their vote on the mailing list. Only maintainers
+of the repository that the candidate is proposed for are allowed to vote. The
+BDFL is always allowed to vote, and has a veto.
+
+If a candidate is approved, a maintainers will contact the candidate to
+invite the candidate to open a pull request that adds the contributor to
+the "maintainers" file. The candidate becomes a maintainer once the pull
+request is merged.
+"""
+
+    [Rules.stepping-down-policy]
+
+        title = "Stepping down policy"
+
+        text = """
+Life priorities, interests, and passions can change. If you're a maintainer but
+feel you must remove yourself from the list. Inform other maintainers that you
+intend to step down, and if possible, help find someone to pick up your work.
+At the very least, ensure your work can be continued where you left off.
+
+After you've informed other maintainers, create a Pull Request to remove
```

Comment:
No need to capitalize **P**ull **R**equest (we don't do it elsewhere in the file it seems).